### DISCUSSION TITLE
Plans for production version for 8.2.0-beta1

### DISCUSSION THREAD

**Discussion by AppSphere** [rnmapbox/maps#1287] [General]
Hello Team,

Thanks for providing a wonderful react native integration with Map Box. I need to apply the production fix for 8.2.0-beta1 in mt production but as per our internal rules we do not use the Beta Build in Production, do we have any plans to push this Beta into actual release?

Regards.
A

**Comment by mfazekas** [comment]
@AppSphere sorry we don't have ETA on when we're publishing a final release.
